kumascript/macros/ReadOnlyInline.ejs

```diff
@@ -1,34 +1,74 @@
-        break;
-    case "fr":
-        str = "Lecture seule ";
-        title = "Cette valeur ne peut pas être changée.";
-        break;
-    case "ja":
-        str = "読取専用 ";
-        break;
-    case "pt-BR":
-        str = "Somente leitura ";
-        title = "Este valor não pode ser alterado.";
-        break;
-    case "ru":
-        str = "Только для чтения ";
-        title = "Это значение изменять нельзя.";
-        break;
-    case "zh-CN":
-        str = "只读 ";
-        title = "该属性的值无法更改";
-        break;
-    default:
-        break;
-}
+var str = mdn.localString({
+    "ca": "Read only ",
+    "cs": "Read only ",
+    "de": "Schreibgeschützt ",
+    "el": "Read only ",
+    "es": "Read only ",
+    "fa": "Read only ",
+    "fi": "Read only ",
+    "fr": "Lecture seule ",
+    "he": "Read only ",
+    "hr": "Read only ",
+    "hu": "Read only ",
+    "id": "Read only ",
+    "it": "Read only ",
+    "ja": "読取専用 ",
+    "ka": "Read only ",
+    "ko": "Read only ",
+    "ms": "Read only ",
+    "nl": "Read only ",
+    "pl": "Read only ",
+    "ro": "Read only ",
+    "ru": "Только для чтения ",
+    "sq": "Read only ",
+    "th": "Read only ",
+    "tr": "Read only ",
+    "vi": "Read only ",
+    "bn-BD": "Read only ",
+    "fy-NL": "Read only ",
+    "ga-IE": "Read only ",
+    "pt-BR": "Somente leitura ",
+    "pt-PT": "Read only ",
+    "zh-CN": "只读 ",
+    "zh-TW": "Read only ",
+    "en-US": "Read only "
+});
 
+var title = mdn.localString({
+    "ca": "This value may not be changed.",
+    "cs": "This value may not be changed.",
+    "de": "Dieser Wert kann nicht verändert werden.",
+    "el": "This value may not be changed.",
+    "es": "This value may not be changed.",
+    "fa": "This value may not be changed.",
+    "fi": "This value may not be changed.",
+    "fr": "Cette valeur ne peut pas être changée.",
+    "he": "This value may not be changed.",
+    "hr": "This value may not be changed.",
+    "hu": "This value may not be changed.",
+    "id": "This value may not be changed.",
+    "it": "This value may not be changed.",
+    "ja": "This value may not be changed.",
+    "ka": "This value may not be changed.",
+    "ko": "This value may not be changed.",
+    "ms": "This value may not be changed.",
+    "nl": "This value may not be changed.",
+    "pl": "This value may not be changed.",
+    "ro": "This value may not be changed.",
+    "ru": "Это значение изменять нельзя.",
+    "sq": "This value may not be changed.",
+    "th": "This value may not be changed.",
+    "tr": "This value may not be changed.",
+    "vi": "This value may not be changed.",
+    "bn-BD": "This value may not be changed.",
+    "fy-NL": "This value may not be changed.",
+    "ga-IE": "This value may not be changed.",
+    "pt-BR": "Este valor não pode ser alterado.",
+    "pt-PT": "This value may not be changed.",
+    "zh-CN": "该属性的值无法更改",
+    "zh-TW": "This value may not be changed.",
+    "en-US": "This value may not be changed."
+});
```

Comment:
You can then do the same cleanup here. Thank you!

Reply:
Alright, that should do it.